docs/apidiffs/current_vs_latest/opentelemetry-sdk-common.txt

```diff
@@ -1,2 +1,8 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
++++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) io.opentelemetry.sdk.common.RandomHolder  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.common.RandomHolder androidFriendly()
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.common.RandomHolder create(java.util.Random)
+	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.Random getRandom()
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.common.RandomHolder platformDefault()
```

Comment:
do we really need these to be added to the public API? I think if they need to be accessed outside the package, we should definitely make them `internal`

Reply:
Moved :)

Reply:
I somehow caught github in some weird temporary state where this was showing. sorry for the noise.

Reply:
You started reviewing as I was pushing changes :)  Not noise, you literally commented as I was fixing locally.  Concurrency at its finest.